packages/amplify/amplify_flutter_ios/example/pubspec.yaml

```diff
@@ -0,0 +1,43 @@
+name: amplify_flutter_ios_example
+description: Test bed for amplify_flutter_ios
+
+publish_to: "none" # Remove this line if you wish to publish to pub.dev
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=2.2.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  amplify_flutter_ios:
+    path: ../
+
+  cupertino_icons: ^1.0.2
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^1.0.0
+  amplify_analytics_pinpoint:
+    path: ../../../analytics/amplify_analytics_pinpoint
```

Comment:
Are all of these imports necessary for the amplify_flutter_ios and amplify_flutter_android example apps?  I thought they were just needed in the main amplify_flutter example app.

Reply:
Thanks, good suggestion! Removed this here and in android version, much simpler.